app/models/project.rb

```diff
@@ -22,6 +22,9 @@ class Project < ApplicationRecord
   TYPE_TASK = { project_type: 0, test_type: 1 }.freeze
   enum project_type: TYPE_TASK
 
+  STATUSES = { active: 0, notification_sent: 1, completed: 2, favorite: 3 }.freeze
+  enum status: STATUSES
```

Comment:
![image](https://user-images.githubusercontent.com/36412453/67249586-e19a2180-f470-11e9-8270-669667234441.png)

i suggest need add sufix status to enum
enum status: STATUSES, , _suffix: true

Reply:
when you add suffix, need to rename all statuses where they used

Reply:
Good idea, i added prefix for more readable code